src/cli.ts

```diff
@@ -169,5 +176,111 @@ program
 				console.error('Invalid action. Use "getip" or "setip".');
 		}
 	});
+program
+	.command('setbrightness')
+	.description('Send http request for changing brightness.')
+	.argument('<Mode>', 'Set to either Enabled or Disabled.')
```

Comment:
If the options here are `enabled` or `disabled`, we should limit that with `.choices`, right?  Also, are we sure it has to be capital or lowercase letters?  Guessing the API is picky :)